<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="SAJIDs-MBP.fios-router.home" name="testNGListeners.SubBaseTest" tests="7" failures="1" timestamp="2021-06-04T00:04:37 EDT" time="0.009" errors="0">
  <testcase name="mercuryTest" time="0.007" classname="testNGListeners.SubBaseTest">
    <failure type="java.lang.AssertionError" message="expected [33] but found [22]">
      <![CDATA[java.lang.AssertionError: expected [33] but found [22]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at testNGListeners.SubBaseTest.mercuryTest(SubBaseTest.java:14)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- mercuryTest -->
  <system-out/>
  <testcase name="venusTest" time="0.000" classname="testNGListeners.SubBaseTest">
    <skipped/>
  </testcase> <!-- venusTest -->
  <system-out/>
  <testcase name="marsTest" time="0.001" classname="testNGListeners.SubBaseTest"/>
  <system-out/>
  <testcase name="jupiterTest" time="0.000" classname="testNGListeners.SubBaseTest"/>
  <system-out/>
  <testcase name="saturnTest" time="0.000" classname="testNGListeners.SubBaseTest"/>
  <system-out/>
  <testcase name="uranusTest" time="0.000" classname="testNGListeners.SubBaseTest"/>
  <system-out/>
  <testcase name="neptuneTest" time="0.001" classname="testNGListeners.SubBaseTest"/>
  <system-out/>
</testsuite> <!-- testNGListeners.SubBaseTest -->
